pkgs/code_assets/doc/linking_issues.md

```diff
@@ -0,0 +1,44 @@
+# Linking Issues with Native Assets
+
+This document describes common linking issues that can occur when using native
+assets in Dart and provides workarounds.
+
+## Conflicting with Embedder Library Dependencies
+
+When a native asset provides a library that is also a dependency of the embedder
+(e.g., a Flutter app), you may encounter runtime crashes. This is because the
```

Comment:
nit: I find the "(e.g., a Flutter app)" kind of confusing here and disrupting the reading flow. I'd probably remove it altogether. It adds very little. The next paragraph then introduces Flutter as a potential embedder.